jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererConstants.kt

```diff
@@ -39,6 +39,12 @@ object CodeWhispererConstants {
     const val USING: String = "using"
     const val GLOBAL_USING: String = "global using"
     const val STATIC: String = "static"
+    const val REQUIRE: String = "require"
+    const val REQUIRE_RELATIVE: String = "require_relative"
+    const val LOAD: String = "load"
+    const val INCLUDE: String = "include"
+    const val EXTEND: String = "extend"
+    const val AS: String = " as "
```

Comment:
nit: I feel this list will keep growing, maybe we can consider move these syntax keywords to a different constant file

Reply:
Yes, I agree, I am working on this, to raise a PR after this PR, with optimization in this constants file, implementation and testing files of security scans as their is some duplicate code.

Made a note and I will move these constant strings to separate file.